src/waveresponse/_standardized1d.py

```diff
@@ -0,0 +1,397 @@
+
+    def _sigma(self, omega_p, sigma_a, sigma_b):
+        """
+        Spectral width.
+        """
+        arg = self._freq <= omega_p
+        sigma = np.empty_like(self._freq)
+        sigma[arg] = sigma_a
+        sigma[~arg] = sigma_b
+        return sigma
+
+
+class OchiHubble(BaseSpectrum1d):
+    """
+    Ochi-Hubble wave spectrum, given as:
+
+    ``S(w) = C / w ** (4 * q + 1) * exp(-d / w ** 4)``
+
+    where,
+
+    - ``C = (1.0 / 4.0) * (c ** q * Hs ** 2) / gamma(q)``
+    - ``c = (4.0 * q + 1.0) * w_p ** 4 / 4.0``
+    - ``d = (4 * q + 1) * w_p ** 4 / 4.0``
+    - ``Hs`` is the significant wave height.
+    - ``w_p = 2pi / Tp`` is the angular spectral peak frequency.
+
+    and ``q`` is a user-defined shape parameter. Note that ``gamma`` is the "Gamma function".
+
+    Parameters
+    ----------
+    freq : array-like
+        Sequence of frequencies to use when generating the spectrum.
+    freq_hz : bool
+        Whether the provided frequencies are in rad/s (default) or Hz.
+
+    Notes
+    -----
+    The special case ``q=1`` corresponds to the modified Pierson-Moskowitz spectrum.
+
+    See Also
+    --------
+    ModifiedPiersonMoskowitz : Modified Pierson-Moskowitz wave spectrum.
+    JONSWAP : JONSWAP wave spectrum.
+    """
+
+    def __call__(self, hs, tp, q=2, freq_hz=None):
+        """
+        Generate wave spectrum.
+
+        Parameters
+        ----------
+        hs : float
+            Significant wave height, Hs.
+        tp : float
+            Peak period, Tp.
+        q : float
+            Spectral shape parameter.
+        freq_hz : bool, optional
+            Whether to return the frequencies and spectrum in terms of rad/s (`True`)
+            or Hz (`False`). If `None` (default), the original units of `freq` is
+            preserved.
+
+        Return
+        ------
+        freq : 1-D array
+            Frequencies corresponding to the spectrum values. Unit is set according
+            to `freq_hz`.
+        spectrum : 1-D array
+            Spectrum values. Unit is set according to `freq_hz`.
+
+        Notes
+        -----
+        The scaling between wave spectrum in terms of Hz and rad/s is defined
+        as:
+
+        ``S(f) = 2*pi*S(w)``
+
+        where ``S(f)`` and ``S(w)`` are the same spectrum but expressed
+        in terms of Hz and rad/s, respectively.
+        """
+
+        return super().__call__(hs, tp, q=q, freq_hz=freq_hz)
+
+    def _spectrum(self, omega, hs, tp, q):
+        args = (hs, tp, q)
+        C = self._C(*args)
+        d = self._d(*args)
+
+        return C / omega ** (4 * q + 1) * np.exp(-d / omega**4)
+
+    def _C(self, *args):
+        hs, tp, q = args
+        omega_p = 2.0 * np.pi / tp
+        c = (4.0 * q + 1.0) * omega_p**4 / 4.0
+        return (1.0 / 4.0) * (c**q * hs**2) / gammafun(q)
+
+    def _d(self, *args):
+        _, tp, q = args
+        omega_p = 2.0 * np.pi / tp
+        return (4 * q + 1) * omega_p**4 / 4.0
```

Comment:
Same here, why use *args. Shouldnt we be more explicit.

Reply:
Fixed.